mlflow/errors.go

```diff
@@ -0,0 +1,20 @@
+package mlflow
+
+const (
+	// ErrorResourceAlreadyExists indicates that a resource with the given name already exists.
+	ErrorResourceAlreadyExists = "RESOURCE_ALREADY_EXISTS"
```

Comment:
Did you mean to use these `const`s ?
Maybe as the type of `ErrorCode`?
Don't see any reference to them.

Reply:
This is intentional.
There might be additional errors we're not aware of, so I kept the error code as a simple string.
There are just for users of the SDK to compare to.